<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimCtrlPClass</class>
 <widget class="QWidget" name="SimCtrlPClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>382</width>
    <height>258</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SimCtrlP</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>231</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="fieldGrowthPolicy">
     <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="displayLab">
      <property name="text">
       <string>display</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="dispMode"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>0</y>
     <width>121</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="startSim">
      <property name="text">
       <string>start</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pauseSim">
      <property name="text">
       <string>pause</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="stopSim">
      <property name="text">
       <string>stop</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="exportPSH">
      <property name="text">
       <string>Export PSH</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="exportSim">
      <property name="text">
       <string>Export sim state</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>191</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QCheckBox" name="calcSRHist">
      <property name="text">
       <string>Calculate SR histograms</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="dispPSH">
      <property name="text">
       <string>Calculate/Display PSH</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="dispRaster">
      <property name="text">
       <string>Display activity raster</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="dispGRGOActs">
      <property name="text">
       <string>Display GR GO activity</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="dispSpikeRates">
      <property name="text">
       <string>Display spikeRates</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="formLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_2">
    <item row="0" column="0">
     <widget class="QSpinBox" name="mZNumBox"/>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Micro Zone number</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>startSim</sender>
   <signal>clicked()</signal>
   <receiver>SimCtrlPClass</receiver>
   <slot>startSim()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>362</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>229</x>
     <y>27</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pauseSim</sender>
   <signal>clicked()</signal>
   <receiver>SimCtrlPClass</receiver>
   <slot>pauseSim()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>256</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>198</x>
     <y>42</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stopSim</sender>
   <signal>clicked()</signal>
   <receiver>SimCtrlPClass</receiver>
   <slot>stopSim()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>59</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dispMode</sender>
   <signal>activated(int)</signal>
   <receiver>SimCtrlPClass</receiver>
   <slot>changeDispMode(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>84</x>
     <y>11</y>
    </hint>
    <hint type="destinationlabel">
     <x>57</x>
     <y>31</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dispPSH</sender>
   <signal>stateChanged(int)</signal>
   <receiver>SimCtrlPClass</receiver>
   <slot>changePSHMode(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>138</y>
    </hint>
    <hint type="destinationlabel">
     <x>104</x>
     <y>47</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dispSpikeRates</sender>
   <signal>clicked()</signal>
   <receiver>SimCtrlPClass</receiver>
   <slot>dispSpikeRates()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>148</x>
     <y>240</y>
    </hint>
    <hint type="destinationlabel">
     <x>226</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exportPSH</sender>
   <signal>clicked()</signal>
   <receiver>SimCtrlPClass</receiver>
   <slot>exportPSH()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>296</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>237</x>
     <y>63</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dispGRGOActs</sender>
   <signal>stateChanged(int)</signal>
   <receiver>SimCtrlPClass</receiver>
   <slot>changeActMode(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>111</x>
     <y>198</y>
    </hint>
    <hint type="destinationlabel">
     <x>333</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exportSim</sender>
   <signal>clicked()</signal>
   <receiver>SimCtrlPClass</receiver>
   <slot>exportSim()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>263</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel">
     <x>226</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dispRaster</sender>
   <signal>stateChanged(int)</signal>
   <receiver>SimCtrlPClass</receiver>
   <slot>changeRasterMode(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>46</x>
     <y>168</y>
    </hint>
    <hint type="destinationlabel">
     <x>11</x>
     <y>109</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>calcSRHist</sender>
   <signal>stateChanged(int)</signal>
   <receiver>SimCtrlPClass</receiver>
   <slot>changeSRHistMode(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>11</x>
     <y>54</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mZNumBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>SimCtrlPClass</receiver>
   <slot>changeMZDispNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>46</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>312</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>startSim()</slot>
  <slot>pauseSim()</slot>
  <slot>stopSim()</slot>
  <slot>changeDispMode(int)</slot>
  <slot>changePSHMode(int)</slot>
  <slot>dispSpikeRates()</slot>
  <slot>exportPSH()</slot>
  <slot>changeActMode(int)</slot>
  <slot>exportSim()</slot>
  <slot>changeRasterMode(int)</slot>
  <slot>changeSRHistMode(int)</slot>
  <slot>changeMZDispNum(int)</slot>
 </slots>
</ui>
